api/src/main/java/io/opentelemetry/trace/samplers/SimpleDecision.java

```diff
@@ -22,7 +22,7 @@
 import java.util.Map;
 
 /** Sampling decision without attributes. */
-final class DefaultDecision implements Decision {
+final class SimpleDecision implements Decision {
```

Comment:
Immutable tag?

Reply:
Well there are no tags at all :)

Reply:
`@Immutable` for the class.

Reply:
I think it is called annotation probably